lib/lambda_ethereum_consensus/state_transition/operations.ex

```diff
@@ -174,7 +174,56 @@ defmodule LambdaEthereumConsensus.StateTransition.Operations do
     Enum.reverse(withdrawals)
   end
 
-  @doc """
+  @spec process_attester_slashing(BeaconState.t(), SszTypes.AttesterSlashing.t()) ::
+          {:ok, BeaconState.t()} | {:ok, nil}
+  def process_attester_slashing(state, attester_slashing) do
+    attestation_1 = attester_slashing.attestation_1
+    attestation_2 = attester_slashing.attestation_2
+
+    cond do
+      Predicates.is_slashable_attestation_data(attestation_1.data, attestation_2.data) == false ->
```

Comment:
nit. Same for other checks
```suggestion
      not Predicates.is_slashable_attestation_data(attestation_1.data, attestation_2.data) ->
```

Reply:
In first time I used it with not but it didn't work properly that's why I changed to `== false`. Don't know why but it doesn't work this way

Reply:
nvm it was about something else